app/views/bets/_display_bets.html.erb

```diff
@@ -5,10 +5,22 @@
         <tr>
           <th class="col-md-2">Created By</th>
           <th class="col-md-2">Created at</th>
+          <th class="col-md-2">Expires at</th>
           <th class="col-md-2">Total amount bet</th>
-          <th class="col-md-6">Bet description</th>
+          <th class="col-md-4">Bet description</th>
         </tr>
       </thead>
+      <tbody>
+        <% bets.each do |bet| %>
+          <tr>
+            <td class="col-md-2"><%= bet.creator_id %></td>
```

Comment:
another cool rails trick is the `content_tag` view helper. Its not always the right choice, but its a cool 'ruby-esque' way of creating tags.

`<%= content_tag :td, bet.creator_id, class: 'col-md-2' %>`

http://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag